### ISSUE TITLE
TestF1Score by class

### ISSUE THREAD

**Issue by danilyef** [evidentlyai/evidently#669]
Hello dear evidently Team!

I would like to request the possibility to add TestF1Score by class option, if it possible. This is done for Precision and Recall already, but not for F1-Score. Ideally would be nice to have option to chose which f1-score strategy to use ('micro-average','macro-average' etc. Reference: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html) 
Many thanks.

**Comment by elenasamuylova** [comment]
Hi @danilyef, thanks for the feature request!

We currently have:
* `TestF1ByClass(label='classN') `
* `TestF1Score()`

So it is already possible to test F1 by class.

To double-check, you also would like to have a micro/macro parameter for the `TestF1Score()`, correct?
